pinot-core/src/main/java/org/apache/pinot/core/operator/streaming/StreamingResponseUtils.java

```diff
@@ -34,6 +35,12 @@ private StreamingResponseUtils() {
     return getResponse(dataTable, Response.ResponseType.DATA);
   }
 
+  public static Server.ServerResponse getDataResponse(InstanceResponseBlock resultsBlock)
```

Comment:
(major) We should remove the metadata from the data response
(minor) Suggest not introducing `InstanceResponseBlock` into this class, convert the `DataTable` on the caller side instead